ui/page/dcrdex/market.go

```diff
@@ -451,7 +448,7 @@ func (pg *DEXMarketPage) listenForOrderbookNotifications() {
 		select {
 		case <-pg.ctx.Done():
 			return
-		case bookUpdate, ok := <-pg.selectedMarketOrderBook.feed.Next():
```

Comment:
RE: https://github.com/crypto-power/cryptopower/pull/692#issuecomment-2423800125

Yes, this caused the panic since we run `listenForOrderbookNotifications` in a goroutine and still try to access `pg.selectedMarketOrderBook.feed` which might become nil when the page state changes. 

I think this is just the only case where this might happen on this page, but will look again.

Reply:
> but will look again

Yes, that's the only case where we might have an issue.